stock_request/security/stock_request_security.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <data noupdate="0">
```

Comment:
[Remove <data> and </data> in xml files if noupdate="0"](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0)